lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -2071,6 +2151,58 @@ TEE_Result crypto_cipher_init(void *ctx, uint32_t algo,
 	const mbedtls_cipher_info_t *cipher_info = NULL;
 	int lmd_res;
 	int lmd_mode;
+#if defined(CFG_CRYPTO_CTS)
+	struct tee_symmetric_cts *cts;
+#endif
+#if defined(CFG_CRYPTO_XTS)
+	struct tee_symmetric_xts *xts;
+#endif
+	TEE_Result res __maybe_unused;
+	int ltc_res __maybe_unused;
+	int ltc_cipherindex __maybe_unused;
+
+#if defined(CFG_CRYPTO_XTS)
+	if (algo == TEE_ALG_AES_XTS) {
+		res = tee_algo_to_ltc_cipherindex(algo, &ltc_cipherindex);
```

Comment:
Could move these three lines into the other `if (algo == TEE_ALG_AES_XTS)` block below.

Reply:
OK